app/Http/routes.php

```diff
@@ -108,15 +108,15 @@
 
 */
 
-Route::get('api/v1/users/{username}/favourites', [
-    'uses' => 'FavouriteController@getUserFavourites',
-    'as'   => 'getuserfavourite'
-]);
+Route::get('api/v1/users/{username}/favourites', function() {
+    return json_encode(["message" => "allow a user to favourite an episodes"]);
 
-Route::delete('api/v1/users/{username}/favourites', [
-    'uses' => 'FavouriteController@deleteUserFavourites',
-    'as'   => 'getuserfavourite'
-]);
+});
+
+Route::delete('api/v1/users/{username}/favourites', function() {
+    return json_encode(["message" => "allow a user to unfavourite an episodes"]);
+
+});
 
 /*
```

Comment:
**Code Quality - Critical**

There is a lot of repetition. In every route, you are writing `api/v1/sdsd/dsds/`. You can take advantage of the route group and give all the routes a prefix of `api/v1` like so:

```php
Route::group(['prefix' => 'api/v1/'], function() {
    Route::get('users', 'blahblabhblah');
    Route::get('users/{username}/favourites', 'blahblahblah');
});
```